src/pipelines/master_pipeline.py

```diff
@@ -95,41 +95,47 @@ def get_args(
     #FEATURES
     #bag of words
     BOW_APPEND = False,
-    BOW_DIFFERENCE = True,
-    BOW_PRODUCT = True,
-    BOW_COS = True,
-    BOW_TFIDF = True,
+    BOW_DIFFERENCE = False,
+    BOW_PRODUCT = False,
+    BOW_COS = False,
+    BOW_TFIDF = False,
     BOW_VOCAB = None,
-    
+
     #skipthoughts
     ST_APPEND = False,
     ST_DIFFERENCE = False,
     ST_PRODUCT = False,
     ST_COS = False,
-    
+
     #lda
     LDA_APPEND = False,
     LDA_DIFFERENCE = False,
     LDA_PRODUCT = False,
     LDA_COS = False,
-    LDA_VOCAB = 1000,
+    LDA_VOCAB = 5000,
     LDA_TOPICS = 40,
-    
+
+    # wordonehot (will not play nicely with other featurization methods b/c not
+    # vector)
+    WORDONEHOT = False
```

Comment:
To keep Sacred experiments synchronized with our code, WORDONEHOT and WORDONEHOT_VOCAB will need to be added to config in experiments.py